src/routes/about/+page.svelte

```diff
@@ -220,22 +220,22 @@
 						rel="noopener noreferrer">Responsibility-Capability Index</a
 					>, and in the long run based on GDP per capita (similar to Ability to Pay).
 				</QA>
-				<QA question="Why are there different absolute NDC values shown in the country graph?">
-					When NDCs area published by countries, they are expressed in relative terms (e.g. 30 %
+				<QA question="Why are there two absolute NDC values shown in the country graph?">
+					Many countries indicate NDC ambitions in various levels or conditionalities, for example to certain developments
+					such as economic growth, or international support. The two values shown are the minimum and
+					maximum NDC values deduced.
+				</QA>
```

Comment:
We can probably delete this question, right? We decided against having two absolute NDCs in the graph.

```suggestion
```

Reply:
Yes, but I rephrased this question to why there are two values given a single method (the min and max). Or do you think this overcomplexifies the NDC?

Reply:
Ah okay, I didn't read it well enough. I added "sometimes" because not all countries have two dots and rephrased the answer a bit. What do you think? 

```suggestion
				<QA question="Why are there sometimes two absolute NDC values shown in the country graph?">
					Many countries indicate NDC targets at different levels or conditional on, for example, certain developments such as economic growth or international support. The two values shown are the minimum and maximum derived NDC values.
				</QA>
```

Reply:
@MarkMDekker I suggested one more suggestion here that needs to be approved. then we can merge

Reply:
It is only a very small change so if needed we can revert later. But since I will be off for two weeks better if I merge this before. @MarkMDekker